gerrit_to_github_issues/github_issues.py

```diff
@@ -75,3 +75,52 @@ def get_bot_comment(issue: Issue, bot_name: str, ps_number: str) -> IssueComment
     for i in issue.get_comments():
         if i.user.login == bot_name and ps_number in i.body:
             return i
+
+
+def assign_issues(repo)
+    open_issues = [i for i in repo.get_issues() if i.state == 'open']
```

Comment:
This makes me wonder if non-cores have the ability to reopen issues? Might be another command to add if not